packages/app/src/__deprecated__/renderer/locales/default/pl-PL.json

```diff
@@ -1,3 +1,3 @@
 {
   "component.collectingDataModalBody": "I agree on sending anonymized data to Mudita"
```

Comment:
isn't it suppose to be in polish? :D

Reply:
that was synced with phrase, i guess that's some legacy data that still lingers there